CHANGELOG

```diff
@@ -1,6 +1,20 @@
 # ChangeLog
 
-## [Version 1.1] - 2018-03-20
+## [Version 1.2] - 2019-06-18
+
+### Added
+- Clarification that all reviews are, by default fourteen (14) days; and must not be less than seven (7) days, see https://github.com/EclipseFdn/EFSP/issues/14.
```

Comment:
Per the discussion on today's Spec Committee call, I'd like to understand why we're making this requirement more restrictive.  We're trying to entice more projects (ie. MicroProfile) to adopt the EFSP, but if we're making it more restrictive with longer required ballot periods, then that's not going to help.  The original EFSP indicated a minimum review period of one week.  We can clarify that to 7 days.  That would be my preference.

Reply:
I've changed this to a seven day default.